.tool-versions.sample

```diff
@@ -1,3 +1,3 @@
-ruby 3.1.5
+ruby 3.3.1
 nodejs 18.20.4
 python 2.7.18
```

Comment:
I know it's not related, but, why do we need python?

Reply:
I doubt anything modern will need python 2.7.18, maybe you need to try again with 3.